src/main/java/org/kohsuke/github/GHWorkflowRunsIterable.java

```diff
@@ -18,6 +19,15 @@ public GHWorkflowRunsIterable(GHRepository owner, GitHubRequest request) {
         this.request = request;
     }
 
+    public GHWorkflowRunsIterable(GHRepository owner, GitHubRequest.Builder<?> requestBuilder) {
```

Comment:
Please make [this method](https://github.com/hub4j/github-api/blob/bcb71a3c318d243c92b757310cf153b8afc983d5/src/main/java/org/kohsuke/github/GHWorkflowRunQueryBuilder.java#L94-L99) call this constructor and remove the other constructor.

Reply:
Done :+1: